pkg/controller/installation/core_controller.go

```diff
@@ -227,6 +235,12 @@ func add(mgr manager.Manager, r *ReconcileInstallation) error {
 		return fmt.Errorf("tigera-installation-controller failed to watch FelixConfiguration resource: %w", err)
 	}
 
+	// Watch for changes to Windows nodes.
+	err = utils.AddWindowsNodeWatch(c)
```

Comment:
I'm not a fan of this being another Node watch, I'm concerned with large clusters that might have lots of nodes. We already have the 1 watch for the typha Indexer.

Reply:
I think the only way to avoid this would be to have a trigger from the windows upgrade that can trigger the Reconcile.
This is where I think we could make use of the dedupe controller because then we could trigger a Reconcile by sending to the requestChan #1305